src/main/resources/application-dev.yaml

```diff
@@ -1,16 +1,9 @@
-application:
-  group: zagrodnicy (Bycza-zagroda)
-  version: 0.5.0
-
 #Datasource
 spring:
-  banner:
-    location: classpath:banner.txt
   datasource:
     url: jdbc:mysql://${DB_SERVER}:${DB_PORT}/${DB_NAME}?useUnicode=true&serverTimezone=UTC
     username: ${DB_USER}
     password: ${DB_PASSWORD}
-    driver-class-name: com.mysql.cj.jdbc.Driver
 
 #to zeby wyrzucaly sie castomowe wyjatki gdy nie ma endointu jeszcze
```

Comment:
It seems to me that it would be better to use comments in English in the code.

Reply:
I removed the unnecessary comment